ui/desktop/tests/e2e/settings.spec.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+/* eslint-disable no-undef */
+const { test } = require('./fixtures/baseTest');
+const { expect } = require('@playwright/test');
+
+test.describe.fixme('Settings Page tests', async () => {
```

Comment:
Are we using `fixme` here because we are still building out this page and it is in an llb?

Reply:
Yup exactly, this is targeting main so settings won't exist yet. When the LLB gets rebased on top of main we can add tests then.